bot.js

```diff
@@ -17,23 +22,50 @@ let manual
 client.on("ready", () => console.log('¡Team Roulette iniciado!'))
 
 client.on("message", message => {
-    const command =message.content, roulette = new Roulette(data.names)
-    if (message.author.bot || !message.content.startsWith(prefix)) return
+
+    //This could be better here so we make the validation before the initialization of 'args' and 'command' vars
+    
+    if (message.author.bot) return;
+    if (message.content.indexOf(prefix) !== 0) return;
```

Comment:
Sure, this validations are in the correct place. But, is better use a once line. Like this:
```javascript
if (message.author.bot || message.content.indexOf(prefix) !== 0) return ;
```
or the original:  
```javascript
if (message.author.bot || !message.content.startsWith(prefix)) return
```